docs/README.md

```diff
@@ -19,7 +19,7 @@ and looking at the content on the GitHub web UI, since GitHub renders
 Markdown.
 
 When editing the dependencies for the Zulip documentation, you'll want
-to edit both the root requirements.txt as well as
+to edit both `requirements/common.txt` as well as
```

Comment:
I just merged 7370fd69d794932cb22b8ac12db798dc04bcac94, which means one now just needs to edit this in requirements/docs.txt (seemed worth just cleaning this up), so we can change this to just explain they should update `requirements/docs.txt`.